### DISCUSSION TITLE
The best way to store rich text content ( Planetscale ) to be consumed in Next.js?

### DISCUSSION THREAD

**Discussion by slowfound** [prisma/prisma#15507] [Q&A] [answered]
Every tutorial I find is storing the content locally in an MDX file. I have a complete full-stack Prisma, Planetscale MySQL, Next.js setup running, I just want to add a writing and displaying blog feature. I don't want to use a ready-made CMS.

**Comment by jonahallibone** [comment, marked as answer]
Many MDX plugins will output AST or a stringified markdown, both of which can be saved in the database and loaded back into the MDX component for display. I'd look into these options. For AST you'd likely save as JSON, for markdown you'd just save as a string.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
